### PR TITLE
Adds instruments to the loadout and also ports some TG in-hand sprites for them.

### PR DESCRIPTION
**Pull request by 4000daniel1**
Hopefully restricting them to off-duties and passengers will prevent them from being all too overused (might'd be a bit weird to bring your old guitar with you to work every single day).

### CODE REVIEW

**Comment by 4000daniel1** [comment]
!review

**Comment by alsoandanswer** [comment]
Not fond of the idea of crew getting instruments.

**Comment by Arrow768** [comment]
It is already possible to order these items via OPs.
I dont see too much point in adding it to the loadout.

**Comment by 4000daniel1** [comment]
> It is already possible to order these items via OPs. I dont see too much point in adding it to the loadout.

Characters who ICly know how to play instruments, would probably already own an instrument. Therefore having to order one from ops feels a bit odd IMO.

For example, let's say a group of characters wanted to form a little band and want to have band practices while they're off duty, I'd say them bringing their own personal instruments with them from the residential deck makes a whole lot more sense than them having to buy new ones from ops every time they want to practice.

**Comment by sadkermit** [comment]
i like this. i have a character who found a guitar in maintenance which they took to their room - it doesnt make sense to keep ordering one from Ops each time i want to do some RP with a guitar in-hand :S

there is an instruments sfx mute button too, for all the people who dont like hearing shitty MIDIfied songs.

**Comment by OolongCow** [comment]
In theory I support the idea since it allows crew to roleplay a bit more, but in reality I dread the inevitable hub players running down the halls playing Despacito.

If they were non-functional it wouldn't even be a question. As-is I can't really say whether I'm for or against it. I don't think without changes that it's the right call.

**Comment by 4000daniel1** [comment]
Went ahead and made a feedback thread for this: https://forums.aurorastation.org/topic/19807-adding-instruments-to-loadout-yay-or-nay/

File: code/modules/client/preference_setup/loadout/items/general.dm [hunk 1]
```diff
@@ -409,3 +409,27 @@
...
+	display_name = "violin"
+	description = "A wooden musical instrument with four strings and a bow, it is quite old."
+	path = /obj/item/device/synthesized_instrument/violin
+	allowed_roles = list("Off-Duty Crew Member", "Passenger")
+
+/datum/gear/trumpet
+	display_name = "Omnitrumpet"
+	description = "The Omnitrumptet series 400 with more than 30 sound samples and fully customizable high fidelity output provides the ultimate means to toot your own horn."
+	path = /obj/item/device/synthesized_instrument/trumpet
+	allowed_roles = list("Off-Duty Crew Member", "Passenger")
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/items/general.dm:435]: These should be rolled together in a selection (see comic and toothbrush selections above)

**Review by Arrow768 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**